tests/logictest/suites/base/06_show/06_0007_show_roles

```diff
@@ -1,13 +1,13 @@
 statement ok
 CREATE ROLE IF NOT EXISTS 'test';
 
-statement query TIII
-SHOW ROLES;
```

Comment:
Can we guarantee the current role is `account_admin`?

What we need to test is if the role test is created.

Reply:
cc @flaneur2020  Need help .

Reply:
what's the user sqlogictest uses?

if it's 'root', it should guarantees the `account_admin` role

Reply:
> what's the user sqlogictest uses?

Maybe the user can't be guaranteed? So we can't guarantees the `account_admin` role?

Reply:
Let's add more role tests later. If we are not sure, suggest skipping this case first.